Update to use maven wrapper

GeeTransit opened the pull request:
address issue https://github.com/OpenLiberty/guides-common/issues/116

gkwan-ibm committed:
Merge pull request #465 from OpenLiberty/staging

Merge staging to prod - Update README.adoc

GeeTransit committed:
Update to use mvnw

GeeTransit committed:
Revert mvnw for test script

GeeTransit committed:
Separate Mac and Linux commands

Cloud hosted guide does not work properly with combined
.mac_section.linux_section roles on code blocks.

GeeTransit committed:
Update remaining instances of mvn into mvnw or Maven

GeeTransit committed:
Fix incorrect schema

Likely copied from another OpenAPI guide

GeeTransit committed:
Fix typo

Extra `cd finish` added by mistake when adding OS specific commands

GeeTransit committed:
Fix incorrect response schema annotation for /api/systems

Should be an array of SystemData, not a single SystemData. Modified the
guide accordingly, and removed the hotspot since the POST endpoint
better showcases its usage.

GeeTransit committed:
Add back mvnw into the finish directory

This is so that we can run the system microservice. We could also put
this into the finish/system directory.

GeeTransit committed:
Reword Linux-only sections to include what to do for Mac/Windows users

AndrewSasmito reviewed (approved):
Successfully tested on Mac and cloud works on certain labs

dependabot[bot] committed:
Bump org.postgresql:postgresql in /finish/module-testcontainers

Bumps [org.postgresql:postgresql](https://github.com/pgjdbc/pgjdbc) from 42.7.5 to 42.7.6.
- [Release notes](https://github.com/pgjdbc/pgjdbc/releases)
- [Changelog](https://github.com/pgjdbc/pgjdbc/blob/master/CHANGELOG.md)
- [Commits](https://github.com/pgjdbc/pgjdbc/compare/REL42.7.5...REL42.7.6)

---
updated-dependencies:
- dependency-name: org.postgresql:postgresql
  dependency-version: 42.7.6
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump org.testcontainers:junit-jupiter in /finish/module-testcontainers

Bumps [org.testcontainers:junit-jupiter](https://github.com/testcontainers/testcontainers-java) from 1.21.0 to 1.21.1.
- [Release notes](https://github.com/testcontainers/testcontainers-java/releases)
- [Changelog](https://github.com/testcontainers/testcontainers-java/blob/main/CHANGELOG.md)
- [Commits](https://github.com/testcontainers/testcontainers-java/compare/1.21.0...1.21.1)

---
updated-dependencies:
- dependency-name: org.testcontainers:junit-jupiter
  dependency-version: 1.21.1
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump io.vertx:vertx-auth-jwt in /finish/module-testcontainers

Bumps [io.vertx:vertx-auth-jwt](https://github.com/eclipse-vertx/vertx-auth) from 4.5.14 to 5.0.0.
- [Commits](https://github.com/eclipse-vertx/vertx-auth/compare/4.5.14...5.0.0)

---
updated-dependencies:
- dependency-name: io.vertx:vertx-auth-jwt
  dependency-version: 5.0.0
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump org.testcontainers:testcontainers in /finish/module-testcontainers

Bumps [org.testcontainers:testcontainers](https://github.com/testcontainers/testcontainers-java) from 1.21.0 to 1.21.1.
- [Release notes](https://github.com/testcontainers/testcontainers-java/releases)
- [Changelog](https://github.com/testcontainers/testcontainers-java/blob/main/CHANGELOG.md)
- [Commits](https://github.com/testcontainers/testcontainers-java/compare/1.21.0...1.21.1)

---
updated-dependencies:
- dependency-name: org.testcontainers:testcontainers
  dependency-version: 1.21.1
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump org.junit.jupiter:junit-jupiter in /finish/module-testcontainers

Bumps [org.junit.jupiter:junit-jupiter](https://github.com/junit-team/junit5) from 5.12.2 to 5.13.0.
- [Release notes](https://github.com/junit-team/junit5/releases)
- [Commits](https://github.com/junit-team/junit5/compare/r5.12.2...r5.13.0)

---
updated-dependencies:
- dependency-name: org.junit.jupiter:junit-jupiter
  dependency-version: 5.13.0
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump org.postgresql:postgresql in /finish/module-persisting-data

Bumps [org.postgresql:postgresql](https://github.com/pgjdbc/pgjdbc) from 42.7.5 to 42.7.6.
- [Release notes](https://github.com/pgjdbc/pgjdbc/releases)
- [Changelog](https://github.com/pgjdbc/pgjdbc/blob/master/CHANGELOG.md)
- [Commits](https://github.com/pgjdbc/pgjdbc/compare/REL42.7.5...REL42.7.6)

---
updated-dependencies:
- dependency-name: org.postgresql:postgresql
  dependency-version: 42.7.6
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump org.postgresql:postgresql in /finish/module-jwt

Bumps [org.postgresql:postgresql](https://github.com/pgjdbc/pgjdbc) from 42.7.5 to 42.7.6.
- [Release notes](https://github.com/pgjdbc/pgjdbc/releases)
- [Changelog](https://github.com/pgjdbc/pgjdbc/blob/master/CHANGELOG.md)
- [Commits](https://github.com/pgjdbc/pgjdbc/compare/REL42.7.5...REL42.7.6)

---
updated-dependencies:
- dependency-name: org.postgresql:postgresql
  dependency-version: 42.7.6
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump org.postgresql:postgresql in /finish/module-securing

Bumps [org.postgresql:postgresql](https://github.com/pgjdbc/pgjdbc) from 42.7.5 to 42.7.6.
- [Release notes](https://github.com/pgjdbc/pgjdbc/releases)
- [Changelog](https://github.com/pgjdbc/pgjdbc/blob/master/CHANGELOG.md)
- [Commits](https://github.com/pgjdbc/pgjdbc/compare/REL42.7.5...REL42.7.6)

---
updated-dependencies:
- dependency-name: org.postgresql:postgresql
  dependency-version: 42.7.6
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump com.ibm.websphere.appserver.api:com.ibm.websphere.appserver.api.jwt

Bumps [com.ibm.websphere.appserver.api:com.ibm.websphere.appserver.api.jwt](https://github.com/OpenLiberty/open-liberty) from 1.1.100 to 1.1.101.
- [Release notes](https://github.com/OpenLiberty/open-liberty/releases)
- [Commits](https://github.com/OpenLiberty/open-liberty/commits)

---
updated-dependencies:
- dependency-name: com.ibm.websphere.appserver.api:com.ibm.websphere.appserver.api.jwt
  dependency-version: 1.1.101
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

SiaaG-dev reviewed (approved):
Tested on linux successfully

GeeTransit committed:
Fix commands to work on Windows

Note that `mvnw.cmd` does not work on Powershell (need to do
`.\mvnw.cmd`), so I'm assuming all commands are done in the Command
Prompt.

Also fixed URLs to use double quotes which works on Windows and Unix.

GeeTransit committed:
Clarify that both the inventory and system services should be stopped

GeeTransit reviewed (commented):
Tested successfully on Windows

gkwan-ibm committed:
Update README.adoc

gkwan-ibm committed:
Update README.adoc

gkwan-ibm committed:
Merge pull request #467 from OpenLiberty/dependabot/maven/finish/module-testcontainers/org.postgresql-postgresql-42.7.6

Bump org.postgresql:postgresql from 42.7.5 to 42.7.6 in /finish/module-testcontainers

gkwan-ibm committed:
Merge pull request #468 from OpenLiberty/dependabot/maven/finish/module-testcontainers/org.testcontainers-junit-jupiter-1.21.1

Bump org.testcontainers:junit-jupiter from 1.21.0 to 1.21.1 in /finish/module-testcontainers

gkwan-ibm committed:
Merge pull request #469 from OpenLiberty/dependabot/maven/finish/module-testcontainers/io.vertx-vertx-auth-jwt-5.0.0

Bump io.vertx:vertx-auth-jwt from 4.5.14 to 5.0.0 in /finish/module-testcontainers

gkwan-ibm committed:
Merge pull request #470 from OpenLiberty/dependabot/maven/finish/module-testcontainers/org.testcontainers-testcontainers-1.21.1

Bump org.testcontainers:testcontainers from 1.21.0 to 1.21.1 in /finish/module-testcontainers

gkwan-ibm committed:
Merge pull request #471 from OpenLiberty/dependabot/maven/finish/module-testcontainers/org.junit.jupiter-junit-jupiter-5.13.0

Bump org.junit.jupiter:junit-jupiter from 5.12.2 to 5.13.0 in /finish/module-testcontainers

gkwan-ibm committed:
Merge pull request #472 from OpenLiberty/dependabot/maven/finish/module-persisting-data/org.postgresql-postgresql-42.7.6

Bump org.postgresql:postgresql from 42.7.5 to 42.7.6 in /finish/module-persisting-data

gkwan-ibm committed:
Merge pull request #473 from OpenLiberty/dependabot/maven/finish/module-jwt/org.postgresql-postgresql-42.7.6

Bump org.postgresql:postgresql from 42.7.5 to 42.7.6 in /finish/module-jwt

gkwan-ibm committed:
Merge pull request #474 from OpenLiberty/dependabot/maven/finish/module-securing/org.postgresql-postgresql-42.7.6

Bump org.postgresql:postgresql from 42.7.5 to 42.7.6 in /finish/module-securing

gkwan-ibm committed:
Merge pull request #475 from OpenLiberty/dependabot/maven/finish/system/com.ibm.websphere.appserver.api-com.ibm.websphere.appserver.api.jwt-1.1.101

Bump com.ibm.websphere.appserver.api:com.ibm.websphere.appserver.api.jwt from 1.1.100 to 1.1.101 in /finish/system
